packages/astro/components/ViewTransitions.astro

```diff
@@ -26,13 +27,15 @@ const { fallback = 'animate', handleForms } = Astro.props;
 <meta name="astro-view-transitions-enabled" content="true" />
 <meta name="astro-view-transitions-fallback" content={fallback} />
 {handleForms ? <meta name="astro-view-transitions-forms" content="true" /> : ''}
+{intraPageTransitions && <meta name="astro-view-transitions-intra-page" content="true" />}
 <script>
 	import type { Options } from 'astro:transitions/client';
 	import { supportsViewTransitions, navigate } from 'astro:transitions/client';
 	// NOTE: import from `astro/prefetch` as `astro:prefetch` requires the `prefetch` config to be enabled
+	// @ts-ignore
```

Comment:
Think this is a mistake? If you saw errors in your editor it's probably because you have not build the project since this feature was added.

Reply:
Seems like I'm also getting this locally even after building the project. I don't mind adding the `@ts-ignore` here for now though